### DISCUSSION TITLE
Apache Netbeans 16 doesn't suggest named beans nor its attributes in xhtml files

### DISCUSSION THREAD

**Discussion by sqrha** [apache/netbeans#5163] [Q&A]
When working in Apache Netbeans 16 with xhtml file, it doesn't suggest either a bean nor its attributes. 

![xhtml-no-bean-suggestions](https://user-images.githubusercontent.com/2011326/210006123-34cd84d7-918f-4324-b626-c7d7e76a832c.png)

Piece of bean:

```
import jakarta.enterprise.context.RequestScoped;
import jakarta.inject.Inject;
import jakarta.inject.Named;

@RequestScoped
@Named
public class HelloBean {

    @Inject
    private GreetingService greetingService;

    private String name;
    private String country;
    private String greeting;
... geters and setters etc
```
src/main/resource/META-INF/beans.xml

```
<?xml version="1.0" encoding="UTF-8"?>
<beans xmlns="https://jakarta.ee/xml/ns/jakartaee"
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/beans_4_0.xsd"
       version="4.0" bean-discovery-mode="annotated">
</beans>
```
What am I missing?

Using JakartaEE 10 stack.

When the xhtml is deployed it works as expected, bean and attributes are evaluated properly. It just doesn't work in IDE as I wish it would.

**Comment by blatra** [comment]
I had the same problem with a multi module maven project.
One module contained the xhtml files and another module contained the beans.
In this constellation Netbeans did not suggest the beans while editing the xhtml files.
